contracts/src/AccountRegistry.sol

```diff
@@ -36,14 +132,14 @@ contract AccountRegistry is Initializable, EIP712Upgradeable, Ownable2StepUpgrad
     // accountIndex -> recoveryAddress, used for recovery of accounts
     mapping(uint256 => address) public accountIndexToRecoveryAddress;
 
-    // authenticatorAddress -> [32 bits recoveryCounter][32 bits pubkeyId][192 bits accountIndex]
-    mapping(address => uint256) public authenticatorAddressToPackedAccountIndex;
-
     // accountIndex -> nonce, used for prevent replay attacks on updates to authenticators
-    mapping(uint256 => uint256) public signatureNonces;
+    mapping(uint256 => uint256) public accountIndexToSignatureNonce;
```

Comment:
wondering if we should make this uint192 to additionally avoid confusion with packed account